src/components/profile/ViewOrUploadAvatar.js

```diff
@@ -74,9 +74,9 @@ const ViewOrUploadAvatar = props => {
             </>
           )}
         </Section.Stack>
-        <CustomButton style={styles.doneButton} onPress={goToProfile}>
-          Done
-        </CustomButton>
+        <Section.Stack justifyContent="flex-end" grow style={styles.buttonsRow}>
```

Comment:
if `justifyContent="flex-end" grow` could be applied through the styles - apply them this way